docs/management_api_admission.yml

```diff
@@ -0,0 +1,95 @@
+swagger: '2.0'
+
+info:
+  version: '1'
+  title: Device admission
+  description: |
+    An API for device admission handling. Intended for use by the web GUI.
+
+basePath: '/api/management/v1/admission'
+host: 'docker.mender.io'
+
+schemes:
+  - https
+
+paths:
+  /devices/{id}/status:
+    put:
+      summary: Update the admission status of a selected device
+      description: |
+        Changes the given device's admission status.
+        Valid state transitions:
+        - 'pending' -> 'accepted'
+        - 'pending' -> 'rejected'
+        - 'rejected' -> 'accepted'
+        - 'accepted' -> 'rejected'
+      parameters:
+        - name: Authorization
+          in: header
+          required: true
+          type: string
+          format: Bearer [token]
+          description: Contains the JWT token issued by the User Administration and Authentication Service.
+        - name: id
+          in: path
+          description: Device authentication data set identifier.
+          required: true
+          type: string
+        - name: status
+          in: body
+          description: New status
+          required: true
+          schema:
+            $ref: '#/definitions/Status'
+      responses:
+        200:
+          description: The status of the device authentication data set was successfully updated.
+          schema:
+            $ref: "#/definitions/Status"
+          examples:
+            application/json:
+              status: "accepted"
+        400:
+          description: |
+              The request body is malformed or the state transition is invalid. See error for details.
+          schema:
+            $ref: "#/definitions/Error"
+        404:
+          description: The device authentication data set was not found.
+          schema:
+            $ref: "#/definitions/Error"
```

Comment:
it is possible that this endpoint will return 422 Unprocessable Entity (when user will hit max device count limit)
this should also be documented

Reply:
fixed